xc/common/utils/vivado_create_runme.py

```diff
@@ -65,9 +93,8 @@ def create_runme(f_out, args):
 
 write_checkpoint -force design_{name}.dcp
 write_bitstream -force design_{name}.bit
-save_project_as -force design_{name}.xpr
```

Comment:
Don't remove the XPR, that is needed for simulation in ModelSim.

Reply:
Sure, done